kimera_semantics/include/kimera_semantics/color.h

```diff
@@ -53,6 +53,8 @@ class SemanticLabel2Color {
   // Make these public if someone wants to access them directly.
   ColorToSemanticLabelMap color_to_semantic_label_;
   SemanticLabelToColorMap semantic_label_to_color_map_;
+
+  size_t number_of_colored_labels;
```

Comment:
we use a trailing _ for members of a class.
```suggestion
  size_t number_of_colored_labels_;
```